tests/end-to-end/external-solution-integration/internal/http/cloudevent_client.go

```diff
@@ -4,7 +4,9 @@ import (
 	"context"
 	"time"
 
-	retrygo "github.com/avast/retry-go"
+	"github.com/sirupsen/logrus"
+
+	baseretry "github.com/avast/retry-go"
```

Comment:
(nit) why baseretry? You don't need to specify a name at all. If you do so, you can use it with the name`retry` ;)

Reply:
We have two "retry" packages and this cause an error. I've changed the old alias in all references to "github.com/avast/retry-go" in all files, to have the same  naming convention.

Reply:
i have to agree with @nachtmaar. eg. if we use logrus we also import it as `log` even though a package called `log` already exists. Both provide the same functionality so it is imho even more confusing if they use a different name

Reply:
Yes, it's true, but we have ower own "retry" package which is used as "retry" package name in all our tests (imported in 45 files):
```

	"github.com/sirupsen/logrus"

	baseretry "github.com/avast/retry-go"
	cloudevents "github.com/cloudevents/sdk-go"

	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/retry"
```
This is why in the past, someone else used the alias "retry-go" for all imports of "github.com/avast/retry-go" . I only changed this alias name, based on your suggestion.
As discussed, we have to refactor the "retry" functionality for all tests.